test/decl/class/override.swift

```diff
@@ -414,3 +414,28 @@ open class OpenDerivedFinal : OpenBase {
 open class OpenDerivedStatic : OpenBase {
   override public static func classMethod() {}
 }
+
+// Context: https://github.com/apple/swift/issues/57499
+// FixIt for 'override func' that should be 'override var' and vice-versa
+func overrideInLieu() {
+  
+  class BaseClass {
+    var someValueX: Bool {
+      get { return false }
+      set { }
+    }
+    
+    func someFuncX() -> Int { return 0 }
+  }
+
+  class Inherited: BaseClass {
+    override var someFuncX: Int { // expected-error {{property does not override any property from its superclass}}
+    // expected-note@-1 {{did you mean to override the method 'someFuncX'?}}
```

Comment:
Fix-its have to be validated as well via {{<loc>-<loc>=<new code>}} for example - https://github.com/apple/swift/blob/main/test/Constraints/diagnostics.swift#L98

Reply:
Yeah, there were some things that needed to be fixed as you suggested regarding fix-its, that's why I left it for the end.